sdk/core/core-http/src/credentials/credentials.ts

```diff
@@ -1,4 +1,5 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
maybe `unknown` or a `Record<string, unknown>`? Maybe not worth bothering with

Reply:
Used `unknown` instead.